src/context/LitentryApiContext.tsx

```diff
@@ -26,6 +26,22 @@ export function LitentryApiClientProvider({children}: {children: React.ReactNode
   useEffect(() => {
     const init = async () => {
       const cache = new InMemoryCache();
+      // {
+      //   typePolicies: {
```

Comment:
is this a TODO? if so, let's create an issue otherwise let's clean this up

Reply:
Done!